### PR TITLE
Optimize ViT training performance

### PR DESCRIPTION
**Pull request by sneaxiy**
ViT-B16-384-Finetune performance improvement (N1C8, no gradient accumulation, local_batch_size=16, unit: images/sec ):

| Baseline | FusedMatmulScale | FusedMatmulScale + FLAGS_enable_cublas_tensor_op_math |
|---|---|---|
| 792 | 848 (+7%) | 874 (+10%) |

### CODE REVIEW

**Review by ForFishes — APPROVED** [review]
LGTM
